### PR TITLE
add Person, Student and Teacher classes

### PR DESCRIPTION
**Pull request by disc3110**
added classes for Person and inheritance classes for Student and Teacher

### CODE REVIEW

File: lib/student.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Student < Person
+  def initialize(age, name, parent_permission, classroom: nil)
```
Comment (mikethreels) [inline lib/student.rb]: - [x] the `name` and `parent_permission` arguments should have a default value.

File: lib/teacher.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class Teacher < Person
+  def initialize(age, name, parent_permission, specialization)
```
Comment (mikethreels) [inline lib/teacher.rb]: - [x] the `name` and `parent_permission` arguments should have a default value.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Diego, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omar-labana — APPROVED** [review]
# **Approved ✔️**

Hi @disc3110,

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**